src/repositories/job_applicants.js

```diff
@@ -0,0 +1,86 @@
+const knex = requireKnex();
+const baseRepo = requireUtil("baseRepo");
+
+const create = async (payload) => {
+  try {
+    payload = baseRepo.addCreatedTimestamps(payload);
+    let result = await knex.transaction(async (trx) => {
+      const rows = await trx("job_applicants").insert(payload).returning("*");
+      return rows[0];
+    });
+    return result;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getAllJobApplicants = async (query) => {
+  try {
+    if (query && query.tags && query.tags.length > 0) {
+      var arr = query.tags.split(",");
+      return await getJobApplicantsByTags(arr);
```

Comment:
`getJobApplicantsByTags` where is this method and what is significance of tags

Reply:
Ok, then I'll remove this if condition, we don't need this